core/jvm/src/main/scala/zio/FiberPlatformSpecific.scala

```diff
@@ -29,7 +29,7 @@ private[zio] trait FiberPlatformSpecific {
     new Fiber.Synthetic.Internal[Throwable, A] {
       override def await(implicit trace: Trace): UIO[Exit[Throwable, A]] = ZIO.fromCompletionStage(cs).exit
 
-      def children(implicit trace: Trace): UIO[Chunk[Fiber.Runtime[_, _]]] = ZIO.succeed(Chunk.empty)
+      def children(implicit trace: Trace): UIO[Chunk[Fiber.Runtime[_, _]]] = Exit.emptyChunk
```

Comment:
Unfortunately, we can only use `Exit.emptyChunk` in places that the effect is wrapped within another effect. The reason for this is that many methods on `Exit` are eagerly evaluated and this change might alter the behaviour of user programs. e.g., the running the following code will print `Exit`.

```scala
import zio.*

@main def main = {
  val _ = ZIO.succeed("ZIO").map(println(_))
  val _ = Exit.succeed("Exit").map(println(_))
  ()
}
```

Reply:
Done